src/assets/content.js

```diff
@@ -21,35 +21,38 @@ chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
 });
 
 const urlRegex = /https?:\/\/[^\s]+/;
+let instanceCount = 0;
```

Comment:
I would add a comment explaining why we need this variable. Please do similar thing with |urlRegex|. Thanks!